index.bs

```diff
@@ -146,16 +141,15 @@ spec: webidl
     };
   </pre>
   <p>
-    Each <a>powerful feature</a> has one or more aspects that websites can
-    request permission to access. To describe these requests, each feature
-    defines a subtype of {{PermissionDescriptor}} to be its <a>permission
-    descriptor type</a>.
+    Each <a>powerful feature</a> has one or more <dfn export>aspects</dfn> that
```

Comment:
For `export` to make sense, we'd need to be defining "aspect" for the whole web platform. Is that really what we want to do here?